shared/actions/json/profile.json

```diff
@@ -1,6 +1,7 @@
 {
   "prelude": [
-    "import * as Types from '../constants/types/profile'"
+    "import * as Types from '../constants/types/profile'",
```

Comment:
still needed?

Reply:
? yes it uses More. those types should move out of there but keeping it small for now